src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -60,7 +64,20 @@
 
 	private final ContainerCraftConfirm ccc;
 
-	private final int rows = 5;
+	private int rows = 5;
+	private int maxRows = Integer.MAX_VALUE;
```

Comment:
Having a `rows` and `maxRows` seems to be a bit redundant in their use. Moving it to a getter might also be an idea.